Webpack warning when using @nativescript/biometrics ERROR_CODES enum

When I try to use the ERROR_CODES enum from the @nativescript/biometrics plug-in i always get the following webpack warning that the enum could not be exported:

```
WARNING in ./src/app/modules/auth/pages/log-in-page/log-in-page.component.ts 148:32-51
export 'ERROR_CODES' (imported as 'ERROR_CODES') was not found in '@nativescript/biometrics' (possible exports: BiometricAuth)
 @ ./src/app/modules/auth/auth.routing.ts 2:0-79 9:33-51
 @ ./src/app/modules/auth/auth.module.ts 1:0-51 10:12-29
 @ ./src/app/app.routing.ts 10:28-64
 @ ./src/app/app.module.ts 3:0-49 20:12-28
 @ ./src/main.ts 6:0-45 11:69-78
```

I tried to change the webpack config according to [this issue](https://github.com/NativeScript/nativescript-dev-webpack/issues/1094) but this leads to crashing (`FATAL ERROR: Reached heap limit Allocation failed - JavaScript heap out of memory`) or multiple compilation errors: 
```
ERROR in bundle
Module not found: Error: path argument is not a string

ERROR in tns_modules/inspector_modules
Module not found: Error: path argument is not a string

ERROR in src/app/app.component.html:6:2 - error NG5002: Invalid ICU message. Missing '}'.
```

My webpack config looks like this:

```
  webpack.chainWebpack((config) => {
    config.module
      .rule('typescript')
      .use('ts-loader')
      .tap((options) =>
        webpack.mergeWebpack(options, { transpileOnly: false })
      );
  });

  // Also tried this:

  webpack.chainWebpack((config) => {
    config.module
      .rule('ts')
      .use('ts-loader')
      .tap((options) => ({ ...options, transpileOnly: false }));
  });
```

Version of Plug-In: 
`"@nativescript/biometrics": "^1.0.0",`

Maybe I'm just configuring something wrong. I'm pretty new to NS but appreciate any help! And thanks for providing all these Plug-Ins.

--- a/packages/biometrics/index.android.ts
+++ b/packages/biometrics/index.android.ts
@@ -1,5 +1,6 @@
 import { BiometricIDAvailableResult, ERROR_CODES, BiometricApi, BiometricResult, VerifyBiometricOptions } from './common';
 import { Application, AndroidActivityResultEventData, Utils, AndroidApplication } from '@nativescript/core';
+export * from './common';
 
 declare const com: any;
 

--- a/packages/biometrics/index.ios.ts
+++ b/packages/biometrics/index.ios.ts
@@ -1,5 +1,6 @@
 import { Utils } from '@nativescript/core';
 import { BiometricIDAvailableResult, BiometricResult, ERROR_CODES, BiometricApi, VerifyBiometricOptions } from './common';
+export * from './common';
 
 const keychainItemIdentifier = 'TouchIDKey';
 let keychainItemServiceName = null;

